### PR TITLE
[my] Completed basics lessons and 2 advanced lessons

### PR DESCRIPTION
**Pull request by alkaha**
I've completed basics lessons and 2 advanced lessons.

### CODE REVIEW

File: my/lessons/basics/composition.md [hunk 1]
```diff
@@ -6,7 +6,11 @@ order: 8
 lang: my
 ---
 
+<<<<<<< HEAD
```
Comment (doomspork) [inline my/lessons/basics/composition.md]: @alkaha there appear to be a bunch of merge conflicts in this file, can you look at resolving them?

Comment (alkaha) [inline my/lessons/basics/composition.md]: done. :)

File: _config.yml [hunk 2]
```diff
@@ -20,8 +20,8 @@ description:
   fr: Leçons sur le langage de programmation Elixir
   gr: Μαθήματα προγραμματισμού της γλώσσας Elixir
   it: Lezioni sul linguaggio di programmazione Elixir
-  pl: Lekcje programowania w języku Elixir
   my: Pelajaran mengenai bahasa aturcara Elixir
+  pl: Lekcje programowania w języku Elixir
```
Comment (doomspork) [inline _config.yml]: I don't think we need this change, can you remove this file?

File: my/index.md [hunk 3]
```diff
@@ -14,8 +14,7 @@ _Maklumbalas dan penyertaan anda amat dialu-alukan!_
 
 "Elixir ialah satu bahasa aturcara kefungsian(functional language) yang dinamik direka untuk membina aplikasi yang mudah untuk dinaiktaraf(scalable) dan mudah untuk diselenggara." - [elixir-lang.org](http://elixir-lang.org/)
 
-Elixir leverages the battle tested ErlangVM to build distributed and fault-tolerant systems with low-latency out of the box.
-Elixir meng-leverage ErlangVM untuk membina sistem yang bersifat mudah-agih(distributed) dan berdayatahan(fault-tolerant) serta mempunyai redah-kependaman(low-latency) yang disiap-pasang.
+Elixir menuilkan kelebihan ErlangVM untuk membina sistem yang bersifat mudah-agih(distributed) dan berdayatahan(fault-tolerant) serta mempunyai redah-kependaman(low-latency) yang disiap-pasang.
```
Comment (doomspork) [inline my/index.md]: Some of the other translators have started working on a Glossary for terms like these, if you want to check it out: https://github.com/doomspork/elixir-school/blob/master/GLOSSARY.md

File: my/lessons/advanced/concurrency.md [hunk 4]
```diff
@@ -0,0 +1,175 @@
+---
+layout: page
+title: Keserempakan(Concurrency)
```
Comment (doomspork) [inline my/lessons/advanced/concurrency.md]: I'm concerned these `(...)` sections in the titles will cause issues with the UI and CSS.  Can we remove them @alkaha?  

Perhaps we should move all instances of `something (english)` to the Glossary.  Thought?

**Comment by doomspork** [comment]
This is great @alkaha, thank you so much!  I left some feedback for you, I'd love to get your thoughts.

File: _config.yml (hunk 2 above)
Comment (alkaha) [inline _config.yml]: it was edited due to merge error. how do I remove this file from this commit?

File: my/index.md (hunk 3 above)
Comment (alkaha) [inline my/index.md]: agreed. moving them into glossary is a good idea. I'll work on that next.

**Comment by doomspork** [comment]
Sorry for the delay @alkaha, I've been busy with work.  This looks good, I will merge and send out a tweet!  Thank you very much

**Comment by alkaha** [comment]
thanks for the merge. :)

**Pull request merged**